### PR TITLE
[CI] Remove redundant execution of unittests in CI_AutoParallel

### PR DESCRIPTION
**Pull request by haohongxiang**
### PR types

Bug fixes
### PR changes

Others
### Description

[CI] Remove redundant execution of unittests in CI_AutoParallel (PCard-73145)

### CODE REVIEW

**Review by risemeup1 — DISMISSED** [review]

File: tools/auto_parallel/ci_auto_parallel.sh [hunk 1]
```diff
@@ -65,13 +64,16 @@ for file_name in `git diff --numstat upstream/${AGILE_COMPILE_BRANCH} |awk '{pri
     elif [[ ${file_name##*.} == "md" ]] || [[ ${file_name##*.} == "rst" ]] || [[ ${dir1} == "docs" ]];then
         continue
     else
+        # auto_unit_test中除llama单测外的其他单测，已在流水线PR-CI-Distribute-stable中PR级别监测，这里不再冗余执行，只监测llama单测
```
Comment (tianshuo78520a) [inline tools/auto_parallel/ci_auto_parallel.sh]: 是否改成英文注释更好些？

File: tools/auto_parallel/ci_auto_parallel.sh [hunk 2]
```diff
@@ -65,13 +64,16 @@ for file_name in `git diff --numstat upstream/${AGILE_COMPILE_BRANCH} |awk '{pri
...
         for ((i=0; i<${#target_lists_for_semi_auto_ci[@]}; i++)); do
             if [[ $i != ${test_auto_num} ]] && [[ ${file_item} == *${target_lists_for_semi_auto_ci[i]}* ]];then
                 case_list[${#case_list[*]}]=gpt-3_auto
-                case_list[${#case_list[*]}]=auto_unit_test
+                # case_list[${#case_list[*]}]=auto_unit_test
```
Comment (tianshuo78520a) [inline tools/auto_parallel/ci_auto_parallel.sh]: 还需要保留么？不直接删除

**Review by Liujie0926 — APPROVED** [review]

**Review by tianshuo78520a — APPROVED** [review]

File: tools/auto_parallel/ci_auto_parallel.sh (hunk 1 above)
Comment (haohongxiang) [reply tools/auto_parallel/ci_auto_parallel.sh]: Done

File: tools/auto_parallel/ci_auto_parallel.sh (hunk 2 above)
Comment (haohongxiang) [reply tools/auto_parallel/ci_auto_parallel.sh]: Done

**Review by tianshuo78520a — APPROVED** [review]

**Review by risemeup1 — APPROVED** [review]

**Pull request merged**